docs/api.rst

```diff
@@ -1174,7 +1174,15 @@ Taking advantage of overloaded functions, you can use the same function name for
     select * from directors where film_school_id = $1.id
   $$ stable language sql;
 
-Computed relationships have good performance as they follow the `Inlining conditions for table functions <https://wiki.postgresql.org/wiki/Inlining_of_SQL_functions#Inlining_conditions_for_table_functions>`_.
+.. important::
+
+  There are some caveats when working with computed relationships:
+
+  - Computed relationships may have good performance when they are inlined, that is, when they are created following the `inlining conditions for table functions <https://wiki.postgresql.org/wiki/Inlining_of_SQL_functions#Inlining_conditions_for_table_functions>`_.
+
+  - Currently, only the computed relationships that return a ``SETOF`` can be inlined.
```

Comment:
I wonder if we should add this since it's already included in the inlining guide on the pg wiki.

Reply:
Hmmm, yeah you're right, the condition says:

> The function is declared RETURNS SETOF or RETURNS TABLE

And `RETURNS TABLE` does not work for computed relationships. On the other hand, `RETURNS <tablename>`  does work for computed rels, but they are not inlined.  I'll change the wording there.